packages/cardano-services/test/cli.test.ts

```diff
@@ -2133,7 +2133,34 @@ describe('CLI', () => {
           );
           await serverStarted(apiUrl);
           const headers = { 'Content-Type': 'application/json' };
-          const res = await axios.post(`${apiUrl}/${ServiceNames.StakePool}/health`, { headers });
+          const res = await axios.post(`${apiUrl}/${ServiceNames.Asset}/health`, { headers });
+          expect(res.status).toBe(200);
+        });
+
+        it('asset provider server', async () => {
+          proc = withLogging(
+            fork(
+              exePath,
+              [
+                ...baseArgs,
+                '--api-url',
+                apiUrl,
+                '--postgres-connection-string-handle',
+                postgresConnectionStringHandle,
+                '--handle-policy-ids',
+                HANDLE_POLICY_IDS,
+                '--use-typeorm-asset-provider',
+                'true',
+                '--service-names',
+                // todo: Throws a fetchError when I use ServiceNames.Asset
+                ServiceNames.Handle
+              ],
+              { env: {}, stdio: 'pipe' }
+            )
+          );
+          await serverStarted(apiUrl);
+          const headers = { 'Content-Type': 'application/json' };
+          const res = await axios.post(`${apiUrl}/${ServiceNames.Handle}/health`, { headers });
```

Comment:
This is test for `asset` provider: no need for `--handle-policy-ids`, and `ServiceNames.Handle` is not in scope of this. We can still use `postgresConnectionStringHandle` because database is compatible.
```suggestion
                '--postgres-connection-string-asset',
                postgresConnectionStringHandle,
                '--use-typeorm-asset-provider',
                'true',
                '--service-names',
                ServiceNames.Asset
              ],
              { env: {}, stdio: 'pipe' }
            )
          );
          await serverStarted(apiUrl);
          const headers = { 'Content-Type': 'application/json' };
          const res = await axios.post(`${apiUrl}/${ServiceNames.Asset}/health`, { headers });
```

Reply:
`--postgres-connection-string-asset` Throws an error "error: unknown option '--postgres-connection-string-asset'"

We didn't add `withPostgresOption(..., 'asset')`

Reply:
Fixed on 820adb2

But this fix creates a `FetchError: request to http://localhost:1337/health failed, reason: connect ECONNREFUSED ::1:1337` error